cli/repl.rs

```diff
@@ -57,6 +57,24 @@ impl Repl {
       .map_err(AnyError::from)
 
     // Forward error to TS side for processing
+    // forward error to ts side for processing
```

Comment:
Oops, rogue `gu` movement.

```suggestion
    // Forward error to ts side for processing
```